docs/tutorials/scilla-tutorials/nft-collection/public-resources.md

```diff
@@ -0,0 +1,55 @@
+---
+sidebar_position: 3
+---
+
+# Making resources public
+
+For our example, lets say we have some .mp4 files we want to distribute.
+
+Let's consider some options of where we could host these files.
+
+## Surface Web
+
+AWS, Google, and any Linux web server have the advantage of being easily accessible by the general public and the speeds of fetching an image is relatively quick and easy depending on the network speed of you and the image service provider.
+
+These public environments have the potential to potentially go offline for maintenance or user error. Public servers also have the ability to change what image is shown is also possible by replacing the image on the server providing the content.
+
+Example URL : ```https://mybucket.s3.amazonaws.com/myimage.jpg```
+
+## IPFS
+
+A popular alternative is IPFS. When you add a file to IPFS, your file is split into smaller chunks, cryptographically hashed, and given a unique fingerprint called a content identifier (CID). This CID acts as an permanent record of your file as it exists at that point in time.
+
+When other nodes look up your file, they ask their peer nodes who's storing the content referenced by the file's CID. When they view or download your file, they cache a copy — and become another provider of your content until their cache is cleared.
+
+A node can pin content in order to keep (and provide) it forever, or discard content it hasn't used in a while to save space. This means each node in the network stores only content it is interested in, plus some indexing information that helps figure out which node is storing what.
+
+If you add a new version of your file to IPFS, its cryptographic hash is different, and so it gets a new CID. This means files stored on IPFS are resistant to tampering and censorship — any changes to a file don't overwrite the original, and common chunks across files can be reused in order to minimize storage costs.
+
+Bad URL : ```https://permaweb.eu.org/ipfs/QmSjJGa7zXUbixvYJpgkRkaSCYEBtSwgVtfzkoD3QkSsty```
```

Comment:
Using `https` is not bad since it has its own use case.

https://docs.ipfs.io/how-to/best-practices-for-nft-data/#types-of-ipfs-links-and-when-to-use-them

> HTTP gateways provide interoperability for legacy user-agents that cannot resolve IPFS URIs natively.
> Here's an example: https://dweb.link/ipfs/bafybeigdyrzt5sfp7udm7hu76uh7y26nf3efuylqabf3oclgtqy55fbzdi

Explain when to use what with balanced knowledge.

Reply:
Ah, I'm not saying it's a bad idea to use HTTPS here. I'm rather trying to make the point where you shouldn't define a specific gateway like `https://permaweb.eu.org/ipfs/`, rather, user should specifiy `ipfs://QmSjJGa7zXUbixvYJpgkRkaSCYEBtSwgVtfzkoD3QkSsty` 

I'll make this text more explict so it's less confusing.